atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -312,7 +317,7 @@ private void debugLogStateOfPool() {
             StringBuilder currentState = new StringBuilder();
             currentState.append(
                     String.format("POOL STATUS: Current blacklist = %s,%n current hosts in pool = %s%n",
-                    blacklistedHosts.keySet().toString(), currentPools.keySet().toString()));
+                    blacklist.getBlacklistedHosts().keySet().toString(), currentPools.keySet().toString()));
```

Comment:
Law of demeter: could potentially create `String getBlacklist()` or have `Blacklist.toString()` return this content.